extensions/amp-story/amp-story-ads.md

```diff
@@ -0,0 +1,176 @@
+# Advertising support in AMP Stories 
+
```

Comment:
Suggest you add 1-2  sentences describing the purpose of this document.
For example:

This document describes the principles and practices for including ads in your AMP stories.

Reply:
done